docs/federation.md

```diff
@@ -0,0 +1,116 @@
+# Federation
+
+## Introduction
+
+The DACCS Network (“Network”) is a federated network that consists of independently operated and maintained computing
+infrastructure (“Node”) that deploy the DACCS software stack and connect with one another under the framework laid out
+in this document. While anyone is free is deploy the software stack on their own hardware and use it as they choose, in
+order to join the Network as a Node, the Node needs to provide certain capabilities and minimum operating standards, as
+laid out in this document, and which must be agreed upon by the person(s) responsible for maintaining the Node (“Node
+Administrator”).
+
+## Principles of Federation
+
+The following sections describe the requirements for operating and maintaining a Node.
+
+### Operations
+
+- Deployment of software stack: Each Node should deploy the DACCS software stack. Node Administrators are required to
+  upgrade the software stack to the latest DACCS stack within two weeks of the updates release date.
+- Node maintenance: Node Administrators accept that joining the Network implies that they are responsible for
+  provisioning, funding and maintaining their own Node.
+- Node security: Node Administrators are responsible for maintaining secure access to their node and for coordinating
+  with
+  the hosting institution regarding any security related considerations prior to the deployment of the Node. Admin
+  access
+  to the Node must be restricted to listed Node Administrators.
+- Downtime policy: A Node should strive to maintain downtime of no more than 5%. If a Node needs to be taken offline for
+  scheduled maintenance purposes, the Node Administrator must provide users at least 2-weeks notice.
+- New user signups: Node Administrators are responsible for the signups of users on their own Nodes. All new signup
+  requests should be verified and approved or rejected within 2 business days.
+- Node configuration: The DACCS deployment stack consists of core components and optional components. A Node can enable
+  or
+  disable non-core components as per the decisions of the Node Administrator. However, when a feature/component is to be
+  removed, adequate notice for the change should be given to the Node’s users.
+
+### Support and Documentation
+
+A Node should maintain a public web-page with:
```

Comment:
I think we should remove this requirement. Node information will now be accessible through the website and the content on the site is dictated by what is in the node registry.